collect/collect.go

```diff
@@ -483,7 +483,7 @@ func (i *InMemCollector) redistributeTraces(ctx context.Context) {
 	redistrubutionStartTime := i.Clock.Now()
 
 	defer func() {
-		i.Metrics.Histogram("collector_redistribute_traces_duration_ms", i.Clock.Now().Sub(redistrubutionStartTime).Milliseconds())
+		i.Metrics.Histogram("collector_redistribute_traces_duration_ms", float64(i.Clock.Now().Sub(redistrubutionStartTime).Milliseconds()))
```

Comment:
i just noticed the typo! (redistribution). You didn't introduce it, but if it's local, maybe fix?